.github/workflows/regression-tests.yml

```
@@ -127,6 +127,13 @@ jobs:
       if: startsWith(matrix.os, 'windows')
       run: conda install --yes python=${{matrix.python-version}}
 
+    - name: Ensure correct Python version
+      shell: bash
+      run: |
+        PYTHON_VER=$(python3 -c "import sys; print(f'{sys.version_info.major}.{sys.version_info.minor}')")
```

```suggestion
        PYTHON_VER=$(python -c "import sys; print(f'{sys.version_info.major}.{sys.version_info.minor}')")
```
Just trying something...